### DISCUSSION TITLE
Duda de diseño con CSS

### DISCUSSION THREAD

**Discussion by EAleman8** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#426] [Q&A] [answered]
Que tal!

Jamás había visto CSS y estoy teniendo un problema con la práctica, no logro encontrar una forma de crear y acomodar estos 4 recuadros del lado izquierdo con la imagen del lado derecho en esta parte:

![imagen](https://user-images.githubusercontent.com/99446604/158699753-81a6a355-aa14-4455-95ce-d605c07728ee.png)

He investigado a como he tenido tiempo pero en ningún sitio he podido resolver como hacer ese acomodo, ¿es posible hacerlo con flexbox?, he estado intentado pero nada que me salen esas alineaciones de los recuadros con la imagen a la izquierda y siento que estoy ya bastante atrasado con lo que me he atorado en esa práctica. :l

**Comment by doguedogue** [comment]
Si con flex es "fácil", tu puedes vamos 🚀

**Comment by Ingrzs** [comment, marked as answer]
En un contenedor padre mete 4 hijos esos son las cajitas esas

Al contenedor padre dale un display flex, y tus cajita se ordenaran asi, 
Les das propiedad flex wrap

Al contenedor padre dale un tamaño fijo
O ahi iras alternando hasta donde quedaria de esa forma
